### PR TITLE
XRDDEV-2096 Improve documentation regarding Security Server <-> Information System communication

### PR DESCRIPTION
**Pull request by andresrosenthal**
https://nordic-institute.atlassian.net/browse/XRDDEV-2096

### CODE REVIEW

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 1]
```diff
@@ -177,7 +177,11 @@ Doc. ID: UG-SS
   * [8.2 Displaying and Changing the Members of a Local Group](#82-displaying-and-changing-the-members-of-a-local-group)
   * [8.3 Changing the description of a Local Group](#83-changing-the-description-of-a-local-group)
   * [8.4 Deleting a Local Group](#84-deleting-a-local-group)
-* [9 Communication with the Client Information Systems](#9-communication-with-the-client-information-systems)
+* [9 Communication with Information Systems](#9-communication-with-information-systems)
+  * [9.1 Communicating with security server as a service consumer](#91-communicating-with-security-server-as-a-service-consumer)
```
Comment (petkivim) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md:181]: I would change this to "Communication with Service Consumer Information Systems" to make it more clear. Also, headings should be capitalized - all words are capitalized unless they are conjunctions, articles, or prepositions.

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 2]
```diff
@@ -177,7 +177,11 @@ Doc. ID: UG-SS
...
+  * [9.2 Communicating with security server as a service provider](#92-communicating-with-security-server-as-a-service-provider)
```
Comment (petkivim) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md:182]: I would change this to "Communication with Service Provider Information Systems" to make it more clear. Also, headings should be capitalized - all words are capitalized unless they are conjunctions, articles, or prepositions.

**Review by petkivim — COMMENTED** [review]
The document number and version history should be updated.

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 3]
```diff
@@ -1420,14 +1438,19 @@ The connection method for internal network servers in the **service provider rol
 
 4.  Click on a service code in the table.
     
-5.  In the view that opens, change the protocol in the service URL to HTTP, HTTPS or HTTPS NO AUTH.
-    If the HTTPS protocol was selected, select the **Verify TLS certificate** checkbox if needed (see section [6.6](#66-changing-the-parameters-of-a-service)). According to the service parameters, the connection with the internal network server is created using one the following protocols:
+5.  In the view that opens, change the protocol (a.k.a. scheme) part in the Service URL to either **http://** or **https://**.
```
Comment (petkivim) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md:1441]: I would consider updating this part a little bit:

> 5. In the view that opens, change the protocol (a.k.a. scheme) part in the Service URL to either `http://` or `https://`.
> - HTTP – the service/adapter URL begins with `http://`.
> - HTTPS – the service/adapter URL begins with `https://`.
>    - If **Verify TLS certificate** checkbox is left unchecked it means that service provider information system's TLS certificate is not verified and is trusted by default.
 >   - If **Verify TLS certificate** checkbox is checked it means that service provider information system's TLS certificate is verified. In order to make the information system's TLS certificate trusted, it must be added into security server's **Information System TLS certificate** list (see section [9.3](https://github.com/nordic-institute/X-Road/blob/00c191c0670c231d295beb967826bcaa52bdae38/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#93-managing-information-system-tls-certificates)).
>    - When the service provider information system needs to verify the Security Server's internal TLS certificate, the certificate must be first exported and then imported into the service provider information system's truststore (see section [9.3](https://github.com/nordic-institute/X-Road/blob/00c191c0670c231d295beb967826bcaa52bdae38/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#93-managing-information-system-tls-certificates)).

**Review by petkivim — CHANGES_REQUESTED** [review]

**Comment by andresrosenthal** [comment]
> The changes look good. The only thing that's missing is updating the document number and version history table.

Added

**Review by petkivim — APPROVED** [review]

**Pull request merged**